### DISCUSSION TITLE
Can I ask a question about submitting patches for review?

### DISCUSSION THREAD

**Discussion by quazar3** [AlexandreRouma/SDRPlusPlus#686] [Q&A]
I read this - http://qpleple.com/how-to-contribute-to-a-project-on-github/  - Is this correct?

It doesn’t sound right that you should fork the project?  Is that right? That’s why I thought it would be polite to ask first.  Sorry if I annoyed anyone.  Would like to do a couple small feature adding ones over time that are little pet peeves of mine in the requests.  Really like the program, great job!

**Comment by AlexandreRouma** [comment]
Hello,
I don't accept feature addition patches/pull requests at the moment. I just don't have the time to verify them and the code quality often isn't up to my standards.

I suggest you open an issue with the feature you'd like to have instead.
